database/src/main/scala/cromwell/database/migration/metadata/table/ExecutionTableMigration.scala

```diff
@@ -0,0 +1,269 @@
+package cromwell.database.migration.metadata.table
+
+import cromwell.database.migration.metadata.MetadataCustomSql
+import MetadataCustomSql._
+
+/**
+  * Transform data from the EXECUTION table to metadata
+  */
+class ExecutionTableMigration extends MetadataCustomSql {
+
+  override def queries: Array[String] = {
+    Array(
+      s"""INSERT INTO METADATA_JOURNAL (
+        WORKFLOW_EXECUTION_UUID,
+        METADATA_KEY,
+        CALL_FQN,
+        JOB_SCATTER_INDEX,
+        JOB_RETRY_ATTEMPT,
+        METADATA_VALUE,
+        METADATA_VALUE_TYPE,
+        METADATA_TIMESTAMP
+      )
+        SELECT
+        WORKFLOW_EXECUTION_UUID,
+      'start',
+    CALL_FQN,
+    IDX,
+    ATTEMPT,
+    DATE_FORMAT(e.START_DT, '%Y-%m-%dT%T.%f$Offset'),
+    'string',
+    NOW()
+    FROM EXECUTION e
+    JOIN WORKFLOW_EXECUTION we ON we.WORKFLOW_EXECUTION_ID = e.WORKFLOW_EXECUTION_ID
+    WHERE
+    e.START_DT IS NOT NULL AND
+    e.EXECUTION_ID NOT IN (SELECT EXECUTION_ID FROM TMP_COLLECTOR);""".stripMargin
```

Comment:
ToL these queries seem more interested in finding out what executions *are not* scatters/collectors than which *are* scatters / collectors.  Perhaps there should be a table that contains these executions and then these queries could just join to that.